File: Arrays/1480/1480.c
// This solution is O(n) runtime 

int *runningSum(int* nums, int numsSize, int* returnSize){
    *returnSize = numsSize;
    for(int i = 1; i < numsSize; i++){
        nums[i] += nums[i-1];
    }
    return nums;
}

File: Arrays/1480/1480.py
from typing import List

# This solution is O(n) runtime

class Solution:
    def runningSum(self, nums: List[int]) -> List[int]:
        for i in range(1, len(nums)):
            nums[i] += nums[i-1]
        return nums

sol = Solution()
print(sol.runningSum([1, 3, 5, 7, 9]))